### ISSUE TITLE
Shortcuts

### ISSUE THREAD

**Issue by HenriDellal** [HenriDellal/emerald#22]

**Comment by quite** [comment]
What are these shortcuts? I came here because the shortcut-icons that Termux creates when one drop executable shellscripts in ~/.shortcuts are not displayes in Emerald.

**Comment by quite** [comment]
Ah, likewise it doesn't show add-to-homescreen shortcuts from Firefox. I'm thinking this must all be what this ticket is about :)

**Comment by HenriDellal** [comment]
Yes, you're correct. Shortcuts already work for pre-Oreo devices. There is no support for newer devices yet.
